tests/fgpyo/util/test_metric.py

```diff
@@ -613,11 +613,30 @@ def test_writer_from_str(tmp_path: Path) -> None:
 
     with fpath.open("r") as f:
         assert next(f) == "foo\tbar\n"
+        assert repr(f.newlines) == repr("\r\n")
```

Comment:
nit: I see that you're using this test to also show that the _default_ newlines is `"\r\n"`, but this test is not for that purpose, and so this is a bit of a footgun if someone comes along in the future.  We should at least add a comment, since this would look out of place.  Better, would be to move this into `test_writer_with_kwargs` and parameterize the `lineterminator` in the test.